ci/code_checks.sh

```diff
@@ -105,7 +105,12 @@ if [[ -z "$CHECK" || "$CHECK" == "lint" ]]; then
 
     # Imports - Check formatting using isort see setup.cfg for settings
     MSG='Check import format using isort ' ; echo $MSG
-    isort --recursive --check-only pandas asv_bench
+    ISORT_CMD="isort --recursive --check-only pandas asv_bench"
+    if [[ "$GITHUB_ACTIONS" == "true" ]]; then
+        eval $ISORT_CMD | awk '{print "##[error]" $0}'; $?=${PIPESTATUS[0]}
```

Comment:
```suggestion
        eval $ISORT_CMD | awk '{print "##[error]" $0}'; RET=$(($RET + ${PIPESTATUS[0]}))
```
Assigning to `$?` doesn't seem to work. I think the above is my preferred option. `$RET` will be increased twice, one with 0, the result of the assignment, which feels a bit hacky. But I think other options will make the code more complicated, and the result of the proposed solution should always be correct.